src/pages/public_registration/LogPanel.tsx

```diff
@@ -0,0 +1,80 @@
+import { PublishingTicket, Ticket, TicketStatus } from '../../types/publication_types/ticket.types';
+import { Box, Skeleton, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useQuery } from '@tanstack/react-query';
+import { fetchOrganization } from '../../api/cristinApi';
+import { fetchUser } from '../../api/roleApi';
+import { Registration } from '../../types/registration.types';
+import { getFullName } from '../../utils/user-helpers';
+import { DoiRequestMessagesColumn } from '../messages/components/DoiRequestMessagesColumn';
+import { StyledStatusMessageBox } from '../messages/components/PublishingRequestMessagesColumn';
+import { CompletedPublishingRequestStatusBox } from './action_accordions/CompletedPublishingRequestStatusBox';
+
+interface LogPanelProps {
+  tickets: Ticket[];
+  registration: Registration;
+}
+
+export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
+  const { t } = useTranslation();
+  const ticketStatusesToShow: TicketStatus[] = ['Completed', 'Closed'];
+  const resourceOwnerAffiliationId = registration.resourceOwner.ownerAffiliation;
+  const resourceOwnerId = registration.resourceOwner.owner;
+
+  const organizationQuery = useQuery({
+    enabled: !!resourceOwnerAffiliationId,
+    queryKey: ['organization', resourceOwnerAffiliationId],
+    queryFn: resourceOwnerAffiliationId ? () => fetchOrganization(resourceOwnerAffiliationId) : undefined,
+    meta: { errorMessage: t('feedback.error.get_institution') },
+    staleTime: Infinity,
+    cacheTime: 1_800_000, // 30 minutes
+  });
+
+  const personQuery = useQuery({
```

Comment:
bare for å prøve å være konsekvente med bruken av person når det er Cristin-person, og user når det er NVA-bruker. Men her har vi veldig mange svin på skogen fra før også, så det er sagt 🙃 

```suggestion
  const userQuery = useQuery({
```

Reply:
Godt observert. Henger igjen fra da jeg brukte feil fetch-metode først 😄